src/lib/Tab/Dashboard/EveryoneContentTab.php

```diff
@@ -18,28 +19,34 @@
 
 class EveryoneContentTab extends AbstractTab implements OrderedTabInterface
 {
-    /** @var PagerContentToDataMapper */
+    /** @var \EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper */
     protected $pagerContentToDataMapper;
 
-    /** @var SearchService */
+    /** @var \eZ\Publish\API\Repository\SearchService */
     protected $searchService;
 
+    /** @var \EzSystems\EzPlatformAdminUi\QueryType\ContentSubtreeQueryType */
+    private $contentSubtreeQueryType;
+
     /**
-     * @param Environment $twig
-     * @param TranslatorInterface $translator
-     * @param PagerContentToDataMapper $pagerContentToDataMapper
-     * @param SearchService $searchService
+     * @param \Twig\Environment $twig
+     * @param \Symfony\Contracts\Translation\TranslatorInterface $translator
+     * @param \EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper $pagerContentToDataMapper
+     * @param \eZ\Publish\API\Repository\SearchService $searchService
      */
     public function __construct(
         Environment $twig,
         TranslatorInterface $translator,
         PagerContentToDataMapper $pagerContentToDataMapper,
-        SearchService $searchService
+        SearchService $searchService,
+        ContentSubtreeQueryType $contentSubtreeQueryType
```

Comment:
Better type hint on the interface/abstract class so it's possible to inject different query. Makes it more flexible if anyone needs to change default behavior.